python/pytests/else_test.py

```diff
@@ -63,3 +63,36 @@ def test_else_debug(record_source, golden) -> None:
             }
         )
     )
+
+
+@pytest.fixture(scope="module")
+def record_source_slack() -> kd.sources.JsonlString:
+    content = "\n".join(
+        [
+            """{"text":"Thread 1","user":"UCZ4","time":1,"thread_ts":1,"key":"dev"}""",
```

Comment:
Shouldn't use `"""..."""` here -- those are "docstrings". Lint will likely suggest changing it.